pandas/tseries/resample.py

```diff
@@ -1284,8 +1285,13 @@ def _adjust_dates_anchored(first, last, offset, closed='right', base=0):
     # See https://github.com/pandas-dev/pandas/issues/8683
 
     first_tzinfo = first.tzinfo
+    last_tzinfo = last.tzinfo
```

Comment:
put the issue number as a comment and a brief expl. future readers will appreciate!